### DISCUSSION TITLE
Password protect publicly shared documents

### DISCUSSION THREAD

**Discussion by david-loe** [outline/outline#6620] [Ideas]
I'd like to password protect a publicly shared Document.

**Comment by tommoor** [comment]
I thought we had an issue for this, but I can't find it – converting!

**Reply by EduNetEurope** [reply]
@tommoor Here it is: 
- https://github.com/outline/outline/discussions/2610
Here is another one: 
- https://github.com/outline/outline/issues/6621

Related: 
- https://github.com/outline/outline/discussions/6644
- https://github.com/outline/outline/discussions/6160

**Comment by EduNetEurope** [comment]
@tommoor Any updates on this one?
